### PR TITLE
admin panel (structure only, links not attivated yet)

### PR DESCRIPTION
**Pull request by alecorra**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -197,3 +197,69 @@ p {
     color: #ffffff;
     font-size: 4rem;
 }
+
```
Comment (mporam) [inline css/style.css]: these styles only apply to your admin panel, so take them out of this file and put them into an admin.css file that loads into the admin panel. that way your real portfolio website doesnt have to load them when it doesnt use them

File: adminPanel.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    <a class="admin-home" href="index.html">HOME</a>
+</nav>
+<div class="admin-container">
+    <div class="admin-project-container">
+        <?php
+        if (!empty($result)) {
+            foreach ($result as $img) {
+                echo '<a href="' . $img['url'] . '"><img class="admin-project-link" src="' . $img['project_pic'] . '"></a>';
+                echo '<div class="buttons">';
+                echo '<a class="admin-edit" href="#">EDIT</a>',
```
Comment (mporam) [inline adminPanel.php]: dont comma seperate echos, use multiple echos

File: css/style.css (hunk 1 above)
Comment (alecorra) [reply css/style.css]: ok, i've done. Now admin.css in with the other stylesheet into css folder

File: adminPanel.php (hunk 2 above)
Comment (alecorra) [reply adminPanel.php]: fixed, thanks.

**Pull request merged**